test/built-ins/Iterator/prototype/Symbol.toStringTag/prop-desc.js

```diff
@@ -1,20 +1,18 @@
 // Copyright (C) 2023 Michael Ficarra. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
 /*---
-es6id: 25.1.2.1
+esid: sec-iteratorprototype-@@tostringtag
 description: Property descriptor
 info: |
-    ES6 Section 17
-
-    Every other data property described in clauses 18 through 26 and in Annex
-    B.2 has the attributes { [[Writable]]: true, [[Enumerable]]: false,
-    [[Configurable]]: true } unless otherwise specified.
+  `Iterator.prototype[@@toStringTag]` is an accessor property with attributes { [[Enumerable]]: *false*, [[Configurable]]: *true* }
 features: [iterator-helpers]
 includes: [propertyHelper.js]
 ---*/
-verifyProperty(Iterator.prototype, Symbol.toStringTag, {
-  value: 'Iterator',
-  writable: true,
-  enumerable: false,
-  configurable: true,
-});
+verifyConfigurable(Iterator.prototype, Symbol.toStringTag);
+verifyNotEnumerable(Iterator.prototype, Symbol.toStringTag);
+
+let desc = Object.getOwnPropertyDescriptor(Iterator.prototype, Symbol.toStringTag);
+assert.sameValue(typeof desc.get, 'function');
+assert.sameValue(typeof desc.set, 'function');
+assert.sameValue(desc.value, undefined);
+assert.sameValue(desc.writable, undefined);
```

Comment:
do we really want to have tests for the kind of property when we're hoping to change it?

iow, could we test that it Gets the right value and that writing to a subclass when it's frozen works?

Reply:
I don't think the test262 tests are going to impact our ability to do that, just like the current tests didn't impact our ability to do the accessors. We just revert this PR when we make the change.

Reply:
For me it depends on when we expect to know whether the future change will happen or not. If we expect it relatively soon, I'd prefer to avoid the churn in order to save maintainer time. If it could be indefinitely long, I'd say we should merge this as is.